app/constants/view.js

```diff
@@ -42,7 +42,11 @@ const ViewTypes = keyMirror({
     SET_LAST_CHANNEL_FOR_TEAM: null,
 
     GITLAB: null,
-    SAML: null
+    SAML: null,
+
+    INCREASE_POST_VISIBILITY: null,
```

Comment:
Are we going to want separate actions for increasing visibility up/down from the focused post, or is that out of scope for this?